demos/browser/app/meetingV2/video/VideoTileCollection.ts

```diff
@@ -246,6 +264,59 @@ export default class VideoTileCollection implements AudioVideoObserver {
     }
   }
 
+  collectVideoWebRTCStats(videoMetricReport: { [id: string]: { [id: string]: {} } }): void {
```

Comment:
Isn't this very similar to this `showVideoWebRTCStats` https://github.com/aws/amazon-chime-sdk-js/blob/main/demos/browser/app/meetingV2/video/VideoTileCollection.ts#L226

Reply:
I did not write this myself. I can follow up with some demo app changes after this is released.